src/agor/tools/dev_tools.py

```
@@ -56,6 +56,12 @@
     generate_agent_handoff_prompt,
     generate_mandatory_session_end_prompt,
 )
+from agor.tools.agent_education import (
+    generate_deployment_prompt,
```

Ruff (0.11.9)

`agor.tools.agent_education.generate_deployment_prompt` imported but unused

Remove unused import

(F401)